Solutions/PingFederate/Data Connectors/Connector_CEF_PingFederate.json

```diff
@@ -0,0 +1,120 @@
+{
+    "id": "PingFederate",
+    "title": "PingFederate",
+    "publisher": "Ping Identity",
+    "descriptionMarkdown": "The [PingFederate](https://www.pingidentity.com/en/software/pingfederate.html) data connector provides the capability to ingest [PingFederate events](https://docs.pingidentity.com/bundle/pingfederate-102/page/lly1564002980532.html) into Azure Sentinel. Refer to [PingFederate documentation](https://docs.pingidentity.com/bundle/pingfederate-102/page/tle1564002955874.html) for more information.",
+    "additionalRequirementBanner": "This data connector depends on a parser based on Kusto Function to work as expected. Follow the steps to use this Kusto Function alias **PingFederateEvent** in queries and workbooks. [Follow steps to get this Kusto Function>](https://aka.ms/sentinel-PingFederate-parser)  ",
+    "graphQueries": [
+        {
+            "metricName": "Total data received",
+            "legend": "PingFederate Events",
+            "baseQuery": "PingFederateEvent"
+        }
+    ],
+    "sampleQueries": [
+        {
+            "description" : "Top 10 Rules", 
+            "query": "PingFederateEvent\n | summarize count() by RuleName\n | top 10 by count_"
+        }
+    ],
+    "dataTypes": [
+        {
+            "name": "CommonSecurityLog (PingFederate)",
+            "lastDataReceivedQuery": "PingFederateEvent\n | summarize Time = max(TimeGenerated)\n| where isnotempty(Time)"
+        }
+    ],
+    "connectivityCriterias": [
+        {
+            "type": "IsConnectedQuery",
+            "value": [
+                "PingFederateEvent\n | summarize LastLogReceived = max(TimeGenerated)\n| project IsConnected = LastLogReceived > ago(30d)"
+            ]
+        }
+    ],
+    "availability": {
+        "status": 1

```

Comment:
Status should be 2. Please update the status.

Thanks,
Aditi Maheshwari

Reply:
updated